Fix chat colors and username display

vitvly opened the pull request:
Fixes #6979

Commit:
Fix chat colors and username display; adjust icons

Signed-off-by: Vitaliy Vlasov <siphiuel@gmail.com>

ghost commented:
<!--prchecklist--> 
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | 16d2b0c9 | [#1](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/1/) | 2019-01-14 19:22:44 | ~2 min 43 sec | `ios-e2e` | [:page_facing_up: build log](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/1/consoleText) |
| :heavy_check_mark: | 16d2b0c9 | [#1](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7267/1/) | 2019-01-14 19:34:34 | ~14 min | `android-e2e` | [:package: package](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190114-191958-16d2b0-e2e.apk) |
| :heavy_check_mark: | 16d2b0c9 | [#1](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7267/1/) | 2019-01-14 19:35:21 | ~15 min | `android` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190114-191959-16d2b0-pr.apk) |
| :heavy_check_mark: | 16d2b0c9 | [#1](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/1/) | 2019-01-14 19:39:06 | ~19 min | `macos` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190114-192004-16d2b0-pr.dmg) |
| :heavy_check_mark: | 16d2b0c9 | [#1](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7267/1/) | 2019-01-14 19:44:19 | ~24 min | `ios` | [:package: package](https://i.diawi.com/jkbrjV) |
| :heavy_check_mark: | 16d2b0c9 | [#1](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/1/) | 2019-01-14 19:50:42 | ~30 min | `linux` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190114-192002-16d2b0-pr.AppImage) |
| :heavy_check_mark: | 16d2b0c9 | [#1](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7267/1/) | 2019-01-14 19:54:01 | ~33 min | `windows` | [:package: package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190114-192005-16d2b0-pr.exe) |

jeluard commented on src/status_im/ui/screens/chat/styles/message/message.cljs:
```diff
@@ -49,10 +48,11 @@
   (assoc message-timestamp
          :color
-         (if outgoing colors/blue colors/white)))
+         (if outgoing (if reply? (colors/alpha colors/white 0.7) colors/blue)
```
Could be clearer with a cond?

vitvly replied:
Done.

jeluard commented on src/status_im/ui/screens/chat/styles/message/message.cljs:
```diff
@@ -165,9 +165,14 @@
    :padding-top    6
+   :padding-left   12
+   :padding-right  16
+   :text-align-vertical :center
```
Better to align map values

vitvly replied:
Aligned.

errorists commented:
@siphiuel Hey, some issues I've spotted: 
The formatting on incoming replies looks incorrect, the line below quoted reply is grey, should be 10% black. 
There is a weird white duplicate timestamp in each message 
some of the margins need fixes 
thanks man! if you need any assistance be sure to msg me :)

![screenshot](https://user-images.githubusercontent.com/4932774/51318928-fd911280-1a5b-11e9-852c-1412d95c41c4.png)

rachelhamlin commented:
Hey @siphiuel, are you making the fixes that @errorists pointed out before merging #7304?

vitvly commented:
@rachelhamlin yes, I've switched to TtT screens in the meanwhile, so didn't push the changes yet here. We've been communicating about it with @errorists

vitvly commented:
@errorists finally fixed the margins, ingoing reply line color, and duplicate timestamp thing.

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (49)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | 022c7a61 | [#2](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/2/) | 2019-01-31 14:24:27 | ~15 min | `linux` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/2/consoleText) |
| :heavy_check_mark: | 022c7a61 | [#2](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7267/2/) | 2019-01-31 14:25:44 | ~17 min | `android` | [:package: apk](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190131-140836-022c7a-pr.apk) |
| :heavy_check_mark: | 022c7a61 | [#2](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7267/2/) | 2019-01-31 14:25:59 | ~17 min | `android-e2e` | [:package: apk](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190131-140835-022c7a-e2e.apk) |
| :heavy_check_mark: | 022c7a61 | [#2](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/2/) | 2019-01-31 14:37:06 | ~28 min | `ios-e2e` | [:package: pkg](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190131-140837-022c7a-e2e.app.zip) |
| :heavy_check_mark: | 022c7a61 | [#2](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/2/) | 2019-01-31 14:37:26 | ~28 min | `macos` | [:package: dmg](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190131-140841-022c7a-pr.dmg) |
| :heavy_check_mark: | 022c7a61 | [#2](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7267/2/) | 2019-01-31 14:37:46 | ~29 min | `ios` | [:package: ipa](https://i.diawi.com/XQUUFk) |
| :heavy_check_mark: | 022c7a61 | [#2](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7267/2/) | 2019-01-31 14:44:15 | ~35 min | `windows` | [:package: exe](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190131-140842-022c7a-pr.exe) |
| :heavy_check_mark: | 022c7a61 | [#3](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/3/) | 2019-01-31 15:04:35 | ~20 min | `linux` | [:package: App](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190131-144353-022c7a-pr.AppImage) |
| | | | | | | |
| :x: | e6d82237 | [#3](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/3/) | 2019-01-31 16:31:55 | ~1 min | `macos` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/3/consoleText) |
| :x: | e6d82237 | [#3](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7267/3/) | 2019-01-31 16:32:12 | ~1 min | `ios` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7267/3/consoleText) |
| :x: | e6d82237 | [#3](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/3/) | 2019-01-31 16:32:14 | ~1 min | `ios-e2e` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/3/consoleText) |
| :x: | e6d82237 | [#3](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7267/3/) | 2019-01-31 16:33:49 | ~3 min | `android-e2e` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7267/3/consoleText) |
| :x: | e6d82237 | [#3](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7267/3/) | 2019-01-31 16:33:52 | ~3 min | `android` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7267/3/consoleText) |
| :x: | e6d82237 | [#4](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/4/) | 2019-01-31 16:34:21 | ~4 min | `linux` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/4/consoleText) |
| :x: | e6d82237 | [#3](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7267/3/) | 2019-01-31 16:34:22 | ~3 min | `windows` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7267/3/consoleText) |
| | | | | | | |
| :heavy_check_mark: | 2befc359 | [#4](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/4/) | 2019-02-04 13:42:48 | ~12 min | `macos` | [:package: dmg](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190204-132957-2befc3-pr.dmg) |
| :heavy_check_mark: | 2befc359 | [#4](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/4/) | 2019-02-04 13:44:13 | ~14 min | `ios-e2e` | [:package: pkg](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190204-132954-2befc3-e2e.app.zip) |
| :heavy_check_mark: | 2befc359 | [#4](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7267/4/) | 2019-02-04 13:44:33 | ~14 min | `ios` | [:package: ipa](https://i.diawi.com/VZQQRo) |
| :heavy_check_mark: | 2befc359 | [#4](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7267/4/) | 2019-02-04 13:48:19 | ~18 min | `windows` | [:package: exe](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190204-132958-2befc3-pr.exe) |
| :heavy_check_mark: | 2befc359 | [#4](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7267/4/) | 2019-02-04 13:48:26 | ~18 min | `android` | [:package: apk](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190204-132953-2befc3-pr.apk) |
| :heavy_check_mark: | 2befc359 | [#4](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7267/4/) | 2019-02-04 13:48:30 | ~18 min | `android-e2e` | [:package: apk](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190204-132952-2befc3-e2e.apk) |
| :heavy_check_mark: | 2befc359 | [#5](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/5/) | 2019-02-04 13:55:05 | ~25 min | `linux` | [:package: App](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190204-132956-2befc3-pr.AppImage) |
| | | | | | | |
| :x: | 8c274a82 | [#5](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/5/) | 2019-02-05 15:58:09 | ~11 min | `macos` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/5/consoleText) |
| :heavy_check_mark: | 8c274a82 | [#5](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/5/) | 2019-02-05 16:01:07 | ~14 min | `ios-e2e` | [:package: pkg](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190205-154618-8c274a-e2e.app.zip) |
| :heavy_check_mark: | 8c274a82 | [#5](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7267/5/) | 2019-02-05 16:03:02 | ~16 min | `android-e2e` | [:package: apk](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190205-154616-8c274a-e2e.apk) |
| :heavy_check_mark: | 8c274a82 | [#5](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7267/5/) | 2019-02-05 16:04:34 | ~18 min | `windows` | [:package: exe](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190205-154625-8c274a-pr.exe) |
| :heavy_check_mark: | 8c274a82 | [#5](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7267/5/) | 2019-02-05 16:06:42 | ~20 min | `android` | [:package: apk](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190205-154618-8c274a-pr.apk) |
| :heavy_check_mark: | 8c274a82 | [#6](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/6/) | 2019-02-05 16:07:18 | ~20 min | `linux` | [:package: App](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190205-154623-8c274a-pr.AppImage) |
| | | | | | | |
| :x: | 19a5e918 | [#6](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/6/) | 2019-02-06 14:43:39 | ~14 min | `ios-e2e` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/6/consoleText) |
| :heavy_check_mark: | 19a5e918 | [#6](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/6/) | 2019-02-06 14:44:21 | ~15 min | `macos` | [:package: dmg](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190206-142909-19a5e9-pr.dmg) |
| :heavy_check_mark: | 19a5e918 | [#6](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7267/6/) | 2019-02-06 14:46:26 | ~17 min | `ios` | [:package: ipa](https://i.diawi.com/vvUfA7) |
| :heavy_check_mark: | 19a5e918 | [#6](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7267/6/) | 2019-02-06 14:49:49 | ~20 min | `android` | [:package: apk](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190206-142901-19a5e9-pr.apk) |
| :heavy_check_mark: | 19a5e918 | [#6](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7267/6/) | 2019-02-06 14:52:28 | ~23 min | `android-e2e` | [:package: apk](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190206-142857-19a5e9-e2e.apk) |
| :heavy_check_mark: | 19a5e918 | [#7](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/7/) | 2019-02-06 14:53:24 | ~24 min | `linux` | [:package: App](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190206-142907-19a5e9-pr.AppImage) |
| :heavy_check_mark: | 19a5e918 | [#6](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7267/6/) | 2019-02-06 15:03:21 | ~34 min | `windows` | [:package: exe](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190206-142912-19a5e9-pr.exe) |
| | | | | | | |
| :heavy_check_mark: | a0056b44 | [#7](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/7/) | 2019-02-07 11:02:46 | ~15 min | `ios-e2e` | [:package: pkg](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190207-104706-a0056b-e2e.app.zip) |
| :heavy_check_mark: | a0056b44 | [#7](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/7/) | 2019-02-07 11:03:08 | ~15 min | `macos` | [:package: dmg](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190207-104710-a0056b-pr.dmg) |
| :heavy_check_mark: | a0056b44 | [#7](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7267/7/) | 2019-02-07 11:03:26 | ~16 min | `ios` | [:package: ipa](https://i.diawi.com/pXLHJk) |
| :heavy_check_mark: | a0056b44 | [#7](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7267/7/) | 2019-02-07 11:05:19 | ~18 min | `android` | [:package: apk](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190207-104706-a0056b-pr.apk) |
| :heavy_check_mark: | a0056b44 | [#7](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7267/7/) | 2019-02-07 11:05:21 | ~18 min | `android-e2e` | [:package: apk](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190207-104703-a0056b-e2e.apk) |
| :heavy_check_mark: | a0056b44 | [#7](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7267/7/) | 2019-02-07 11:06:38 | ~19 min | `windows` | [:package: exe](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190207-104711-a0056b-pr.exe) |
| :heavy_check_mark: | a0056b44 | [#8](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/8/) | 2019-02-07 11:15:55 | ~28 min | `linux` | [:package: App](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190207-104709-a0056b-pr.AppImage) |
| | | | | | | |
| :heavy_check_mark: | 1ded37f2 | [#8](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/8/) | 2019-02-08 10:13:05 | ~14 min | `macos` | [:package: dmg](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190208-095828-1ded37-pr.dmg) |
| :heavy_check_mark: | 1ded37f2 | [#8](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/8/) | 2019-02-08 10:13:24 | ~14 min | `ios-e2e` | [:package: pkg](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190208-095826-1ded37-e2e.app.zip) |
| :heavy_check_mark: | 1ded37f2 | [#8](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7267/8/) | 2019-02-08 10:14:18 | ~15 min | `ios` | [:package: ipa](https://i.diawi.com/xvFjhM) |
| :heavy_check_mark: | 1ded37f2 | [#8](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7267/8/) | 2019-02-08 10:15:30 | ~17 min | `android-e2e` | [:package: apk](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190208-095824-1ded37-e2e.apk) |
| :heavy_check_mark: | 1ded37f2 | [#8](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7267/8/) | 2019-02-08 10:17:48 | ~19 min | `windows` | [:package: exe](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190208-095829-1ded37-pr.exe) |
| :heavy_check_mark: | 1ded37f2 | [#8](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7267/8/) | 2019-02-08 10:18:28 | ~20 min | `android` | [:package: apk](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190208-095825-1ded37-pr.apk) |
| :heavy_check_mark: | 1ded37f2 | [#9](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/9/) | 2019-02-08 10:20:52 | ~22 min | `linux` | [:package: App](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190208-095828-1ded37-pr.AppImage) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | c7178a39 | [#9](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/9/) | 2019-02-08 10:27:22 | ~14 min | `macos` | [:package: dmg](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190208-101308-c7178a-pr.dmg) |
| :heavy_check_mark: | c7178a39 | [#9](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/9/) | 2019-02-08 10:28:25 | ~14 min | `ios-e2e` | [:package: pkg](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190208-101326-c7178a-e2e.app.zip) |
| :heavy_check_mark: | c7178a39 | [#9](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7267/9/) | 2019-02-08 10:29:38 | ~15 min | `ios` | [:package: ipa](https://i.diawi.com/2DT9y8) |
| :heavy_check_mark: | c7178a39 | [#9](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7267/9/) | 2019-02-08 10:32:11 | ~16 min | `android-e2e` | [:package: apk](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190208-101536-c7178a-e2e.apk) |
| :heavy_check_mark: | c7178a39 | [#9](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7267/9/) | 2019-02-08 10:34:03 | ~15 min | `android` | [:package: apk](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190208-101832-c7178a-pr.apk) |
| :heavy_check_mark: | c7178a39 | [#9](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7267/9/) | 2019-02-08 10:36:08 | ~18 min | `windows` | [:package: exe](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190208-101758-c7178a-pr.exe) |
| :heavy_check_mark: | c7178a39 | [#10](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/10/) | 2019-02-08 10:45:05 | ~24 min | `linux` | [:package: App](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190208-102059-c7178a-pr.AppImage) |
| | | | | | | |
| :heavy_check_mark: | 7f15f688 | [#10](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7267/10/) | 2019-02-11 12:08:03 | ~14 min | `macos` | [:package: dmg](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190211-115347-7f15f6-pr.dmg) |
| :heavy_check_mark: | 7f15f688 | [#10](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7267/10/) | 2019-02-11 12:08:07 | ~14 min | `ios-e2e` | [:package: pkg](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190211-115344-7f15f6-e2e.app.zip) |
| :heavy_check_mark: | 7f15f688 | [#10](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7267/10/) | 2019-02-11 12:10:30 | ~16 min | `ios` | [:package: ipa](https://i.diawi.com/NBHh8F) |
| :heavy_check_mark: | 7f15f688 | [#10](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7267/10/) | 2019-02-11 12:12:36 | ~18 min | `android-e2e` | [:package: apk](https://status-im.ams3.digitaloceanspaces.com/StatusIm-190211-115343-7f15f6-e2e.apk) |
| :heavy_check_mark: | 7f15f688 | [#10](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7267/10/) | 2019-02-11 12:14:27 | ~20 min | `android` | [:package: apk](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190211-115344-7f15f6-pr.apk) |
| :heavy_check_mark: | 7f15f688 | [#11](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7267/11/) | 2019-02-11 12:16:16 | ~22 min | `linux` | [:package: App](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190211-115346-7f15f6-pr.AppImage) |
| :heavy_check_mark: | 7f15f688 | [#10](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7267/10/) | 2019-02-11 12:25:56 | ~32 min | `windows` | [:package: exe](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-190211-115347-7f15f6-pr.exe) |

errorists commented:
> @errorists finally fixed the margins, ingoing reply line color, and duplicate timestamp thing.

@siphiuel Thanks! 🙏 There's a new issue now, timestamps in outgoing messages are invisible, I guess the white duplicates you fixed were a part of outgoing messages. I also didn't have a chance to test transaction messages thoroughly.

vitvly commented:
@errorists ahh right, fixed the outgoing messages timestamp color (it's 70% white now). Thanks!

churik commented:
@siphiuel can't be built because file is formatted incorrectly. Can you please fix it?
```
(defn message-timestamp-text [justify-timestamp? outgoing rtl?]

   (merge message-timestamp

-         {:color (if outgoing 

+         {:color (if outgoing

                    (colors/alpha colors/white 0.7)

                    colors/gray)}

          (when justify-timestamp? {:position              :absolute

1 file(s) formatted incorrectly

script returned exit code 1
```

vitvly commented:
@churik fixed broken formatting

churik commented:
@siphiuel  following issues should be fixed:
![ready](https://user-images.githubusercontent.com/4557972/52224383-33315a80-28a8-11e9-9979-de89c823e875.png)
![ready](https://user-images.githubusercontent.com/4557972/52224159-9ec6f800-28a7-11e9-8333-e3ac0f65407e.png
![ready 2](https://user-images.githubusercontent.com/4557972/52224381-3298c400-28a8-11e9-927f-3745380711cf.png)
![ready3](https://user-images.githubusercontent.com/4557972/52224380-3298c400-28a8-11e9-8df6-fb8f3ce05075.png)

**(1, 4, 5) No visible border between chat header, menus  and message area - all reproducible for IOS, for Android only 4 can be replicated
(2) No visible currency for incoming `/send ` message
(3) White screen inside incoming `/request` message**

Not sure that designs are updated for these elements.


<blockquote></blockquote>

errorists commented:
thanks @churik!
1) yes, there should be a 1px Light Grey border on the bottom of the Top Bar - it's a wider issue shared with most of the screens in the app. All designs in Figma reflect that as well as our components, however we didn't file an issue for fixing that as we believe it's best resolved on global component level, either the current toolbar component needs updating or it needs to be replaced with a new one cc @yenda 
2) thanks for spotting, the currency string should be using our Grey colour [Figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=3%3A556)
3) the white part looks weird, it shouldn't have a background at all
4) The 1px Light Grey bottom divider is missing on the 'add to contacts' snackbar [Figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=2%3A126)
5) The autocomplete box is missing a shadow `box-shadow: 0px 1px 12px rgba(0, 0, 0, 0.12)` but if it's easier to add then a 1px Light Grey divider on top will also suffice. [Figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=144%3A237)

Most of the above are long overdue issues even before the colour change, if we can update them accordingly it would leave the mobile Chat UI in a much-improved shape. All Figma designs reflect that, lmk if I can be of further help.

vitvly commented:
Items 2,3,4 fixed. For item 5 i've added a light-grey top border, hopefully this will work (did not test on ios)

vitvly commented:
Item 1 - I can see the border on Android:
https://monosnap.com/file/h8xQLQqbmtkrENJEmkMNFNdYofukrp

<blockquote><img src="https://monosnap.com/facebook/generate_image_preview?id=h8xQLQqbmtkrENJEmkMNFNdYofukrp" width="48" align="right"><div>Monosnap screenshot tool</div><div><strong><a href="https://monosnap.com/file/h8xQLQqbmtkrENJEmkMNFNdYofukrp">Screenshot "Android Emulator - Pixel_2_API_275554 2019-02-05 17-48-23.png"</a></strong></div></blockquote>

churik commented:
![sdfsdffddfsdfdfsdf](https://user-images.githubusercontent.com/4557972/52358001-7968f400-2a37-11e9-9b47-2a6089d8a78e.png)


1) @siphiuel  white dot after amount and before currency  (Android only) - shouldn't be here

2) @errorists about  1 - timestamps should be presented for outgoing emojis, yes? They were hardly visible (https://github.com/status-im/status-react/issues/6497); they are not  visible at all after changing background color.
Will be fixed separately or should be here, WDYT?

3)  @errorists elements are not centered inside icons - but I suppose not related to PR, should be reported separately?

Other issues are fixed.

errorists commented:
@churik once again, thanks a bunch for testing 👏 

2) Definitely, they should. correct colour is Grey, similar to incoming emojis. Also it would be nice if the timestamp was placed in the same baseline as emoji, see [this Figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=2%3A541)
I'd say to fix it here in this issue, once pushed we'll have a solid clean chat UI to build further features on
3) there's a couple of things going wrong there, I suspect it's actually caused by the recent PR where Eric updated all icon resources. The text is aligned correctly but the icon is too large and overflows the container, definitely needs to be fixed, I leave it to your judgement when. The correct icon resource should be the `tiny:check` icon, [link to Figma](https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=787%3A1024), and the `tiny pending` icon needs to be added, [link to Figma](https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=1875%3A1797)
For reference, the various transaction dialogues can be found here [Figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=2%3A527)

vitvly commented:
@churik @errorists 
1. Fixed
2. Changed timestamp color to gray and moved it to the same line as the emoji.
3. Added a tiny-pending icon and used both tiny-check and tiny-pending in send-preview.

errorists commented:
@siphiuel you're a legend, thanks! 🙏  one last thing which Eric's PR bugged, the icon for quote bubbles when you're replying to someone, it's not using the `tiny-reply` asset. If you guys recall any other chat specific icons that might have been bugged, pls don't hesitate to fix them, honestly it feels like we're putting down fires caused by his PR xD

yenda commented:
@errorists unfortunately the old icons where not sorted and never named tiny so I missed out on that :(

vitvly commented:
Changed to use tiny-reply instead of main-reply.

errorists commented:
🍾 woohoo, I think this PR is done and ready to be merged friends 👏

rachelhamlin commented:
Nice work! 🤸‍♀️

yenda commented on src/status_im/ui/components/icons/vector_icons.cljs:
```diff
@@ -202,6 +202,7 @@
      ;; Tiny icons
      :tiny-icons/tiny-arrow-down   (components.svg/slurp-svg "./resources/icons/tiny/tiny_arrow_down.svg")
      :tiny-icons/tiny-check        (components.svg/slurp-svg "./resources/icons/tiny/tiny_check.svg")
+     :tiny-icons/tiny-pending      (components.svg/slurp-svg "./resources/icons/tiny/tiny_pending.svg")
```
please alphabetical order and also import for desktop as well just in case

vitvly replied:
Done.
